src/metabase/query_processor/card.clj

```diff
@@ -42,22 +42,20 @@
                   (u/emoji "💾"))
         ttl-seconds))))
 
-(defenterprise db-cache-ttl
-  "Fetches the cache TTL set for the given database. Returns nil on OSS."
+(defenterprise granular-ttl
```

Comment:
is this explicitly returning `nil` in the OSS version? if so we should state that in the docstring as well.